### PR TITLE
Inital port to 1.19 & tried to fix deprecations

### PR DESCRIPTION
**Pull request by QuentiumYT**
Hello,
I recently opened an issue #348 to request an 1.19 version. Trying to understand a bit more java in a abrupt way (no serious knowledge) I was still looking for understanding more Java.

Well as I was aware of, my Java skills are not good enough and couldn't fix all remaining issues (4 left with no real fixes)

So I'm publishing my changes in case it helps you in any way, you can ignore this PR if that's not as good as it should.
There is 2 commit, the first one that is almost sure it works with new API changes, and the second with my attempts to do somthing.

Also, if I could have a few feedbacks of what are my mistakes, I'll take them 😄 

Thanks for your time and kindness.

TLDR: This is not a "please update to 1.19 commit", I'm just interrested if I can help with what I already did

### CODE REVIEW

**Comment by way2muchnoise** [comment]
Thanks for the PR, used this as a starting point for be5d9fbbb346971866a55ee4ce57d533d54ee9d3
